### ISSUE TITLE
Dynamo returns invalid JSON when object contains non-ASCII characters in strings.

### ISSUE THREAD

**Issue by vangorra** [localstack/localstack#464]
Writing the object works, retrieving the object results in an incomplete JSON object being retrieved from dynamo.

The following node code demonstrates the problem:
``` typescript
import * as AWS from 'aws-sdk';

describe('TestSuite', () => {
  it('Test1', async () => {
    AWS.config.dynamodb = {
      accessKeyId: 'KEY_ID',
      secretAccessKey: 'ACCESS_KEY',
      region: 'us-east-1',
      sslEnabled: false,
      endpoint: 'localhost:4569',
    };


    const tableName = 'Table1Test';
    const dynamo = new AWS.DynamoDB({ region: 'us-west-1' });
    const documentClient = new AWS.DynamoDB.DocumentClient({ region: 'us-west-1' });

    // Delete table if it exists.
    try {
      await dynamo.deleteTable({
        TableName: tableName,
      }).promise();
    } catch (e) {}

    // Create the table.
    await dynamo.createTable({
      TableName: tableName,
      ProvisionedThroughput: {
        ReadCapacityUnits: 5,
        WriteCapacityUnits: 5,
      },
      AttributeDefinitions: [
        {
          AttributeName: '_id',
          AttributeType: 'S',
        },
      ],
      KeySchema: [
        {
          AttributeName: '_id',
          KeyType: 'HASH',
        },
      ],
    }).promise();


    const object1 = {
      _id: 'AAAA',
      name: 'Object Name',
      // Get fails with parse error for all the following values.
      // name: 'Object Name…',
      // name: 'Object Name✓',
      // name: 'Object Name¡',
      // name: 'Object Name¢',
      // name: 'Object Name£',
      // name: 'Object Name¤',
      name: 'Object Name¥',
      // name: 'Object Name©',
    };

    // Create the object.
    await documentClient.put({
      TableName: tableName,
      Item: object1,
    }).promise();

    // Get the object. Results in a parse error.
    const request = await documentClient.get({
      TableName: tableName,
      Key: {
        _id: object1._id,
      },
    });

    request.on('httpData', (chunk, response) => {
      console.log('HTTP DATA', chunk.toString());
    });

    await request.promise();
  });
});
```

I haven't tested all characters but the ones listed are a fair sample. The resulting http data returned from dynamo looks like the example below. Notice how the closing } is missing.
```json
{"Item":{"name":{"S":"Object Name¥"},"_id":{"S":"AAAA"}}
```

**Comment by vangorra** [comment]
I confirmed these characters work when using dynamo on AWS. It also works when using AWS' local development dynamodb server.

**Comment by whummer** [comment]
Thanks for reporting @vangorra . This should be fixed in #470
